src/ts/stringutils.ts

```diff
@@ -36,7 +36,7 @@ export namespace StringUtils {
         .replace('ss', leftPadWithZeros(seconds, 2));
   }
 
-  export function secondsToText(totalSeconds: number): string {
+  export function secondsToText(totalSeconds: number, shouldLeftPadNumbers = true): string {
```

Comment:
Having a default padding of 2 feels a bit weird.

Since we only use this value in one place (seekbar.ts) I'd recommend to replace the `shouldLeftPadNumbers` with an attribute that specifies the left pad length, and make it either mandatory or default to `0`.

e.g.
```ts
  export function secondsToText(totalSeconds: number, leftPadLength = 0): string {
```
or
```ts
  export function secondsToText(totalSeconds: number, leftPadLength: number): string {
```

Reply:
I didn't want to change the default. The default is always to have two-digit time strings, such as "06" seconds or "25" minutes etc.
But agree, it might make more sense to provide an actual number as parameter.
I'll update this.

Reply:
Updated in a9594dd75777c648101580700787b80ac9a2c2dc

Reply:
As discussed offline, I've removed the left pad option in 382b2270d97ac753c2124e7f7054b179b5b52441